source/fundamentals/builders/filters.txt

```diff
@@ -11,3 +11,549 @@ Filters Builders
+   { _id: 11, a: 68, binaryValue: "1000100" }
+   { _id: 12, a: 102, binaryValue: "01100110" }
+
+Collection: ``geo_points``
+
+.. code-block:: json
+
+   { _id: 13, "coordinates": { type: "Point", coordinates: [2.0, 2.0] } }
+   { _id: 14, "coordinates": { type: "Point", coordinates: [5.0, 6.0] } }
+   { _id: 15, "coordinates": { type: "Point", coordinates: [1.0, 3.0] } }
+   { _id: 16, "coordinates": { type: "Point", coordinates: [4.0, 7.0] } }
+
+.. _comparison:
+
+Comparison 
+----------
+
+The comparison filters include all operators that compare the value in a
+document to a specified value. 
+
+The comparison operator methods include:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Comparison Method
+     - Matches
+      
+   * - :java-core-api:`eq() <com/mongodb/client/model/Filters.html#eq(java.lang.String,TItem)>`
+     - values equal to a specified value.
+
+   * - :java-core-api:`gt() <com/mongodb/client/model/Filters.html#gt(java.lang.String,TItem)>`
+     - values greater than a specified value.
+      
+   * - :java-core-api:`gte() <com/mongodb/client/model/Filters.html#gte(java.lang.String,TItem)>`
+     - values greater than or equal to a specified value.
+
+   * - :java-core-api:`lt() <com/mongodb/client/model/Filters.html#lt(java.lang.String,TItem)>`
+     - values less than a specified value.
+
+   * - :java-core-api:`lte() <com/mongodb/client/model/Filters.html#lte(java.lang.String,TItem)>`
+     - values less than or equal to a specified value.
+
+   * - :java-core-api:`ne() <com/mongodb/client/model/Filters.html#ne(java.lang.String,TItem)>`
+     - values not equal to a specified value.
+
+   * - :java-core-api:`in() <com/mongodb/client/model/Filters.html#in(java.lang.String,java.lang.Iterable)>`
+     - any of the values specified in an array.
+
+   * - :java-core-api:`nin() <com/mongodb/client/model/Filters.html#nin(java.lang.String,java.lang.Iterable)>`
+     - none of the values specified in an array.
+
+   * - :java-core-api:`empty() <com/mongodb/client/model/Filters.html#empty()>`
+     - all the documents.
+
+The following example creates a filter that matches all documents where
+the value of the ``qty`` field equals "5" in the ``paint_purchases`` collection:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Filters.java
+   :language: java
+   :dedent:
+   :start-after: begin equalComparison
+   :end-before: end equalComparison
+   
+The JSON output renders as the following:
+
+.. code-block:: json
+   :copyable: false
+   
+   { "$eq": { "qty": 5 } }
+
+The following shows the output of the query using the operator specified above:
+
+.. code-block:: json
+   :copyable: false
+   
+   {"_id": 1, "color": "red", "qty": 5, "vendor": ["A"]}
+   {"_id": 6, "color": "pink", "qty": 5, "vendor": ["C"]}
+
+The following example creates a filter that matches all documents where
+the value of the ``qty`` field is greater than or equal to "8" in the
+``paint_purchases`` collection: 
```

Comment:
Nitpick: to keep the result set more compact, we could use 10 instead of 8. Then it would only return two results but still demonstrate values equal and greater, and save a couple lines.